python/cudf/cudf/tests/test_parquet.py

```diff
@@ -2552,9 +2552,9 @@ def test_parquet_reader_zstd_compression(datadir):
         pytest.mark.xfail(reason="zstd support is not enabled")
 
 
-def test_read_parquet_multiple_files(datadir):
-    df_1_path = datadir / "df_1.parquet"
-    df_2_path = datadir / "df_2.parquet"
+def test_read_parquet_multiple_files(tmpdir):
```

Comment:
This shouldn't be a `datadir`, but should be a `tmpdir`. Fixed it in this PR as it is a trivial change.